qlog/src/events/quic.rs

```diff
@@ -787,6 +787,13 @@ pub struct MarkedForRetransmit {
     pub frames: Vec<QuicFrame>,
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Debug)]
+#[serde(untagged)]
+pub enum ConnectionCloseErrorCode {
+    CryptoError(CryptoError),
```

Comment:
The CryptoError type also needs to be updated to reflect the latest qlog spec - https://quicwg.org/qlog/draft-ietf-quic-qlog-quic-events.html#section-8.12.25